ci/infra/testrunner/tests/utils.py

```diff
@@ -81,3 +123,79 @@ def _handle_timeout(signum, frame):
 
     raise Exception("Failed waiting for function {} after {} attemps due to {}".format(func.__name__, attempts, reason))
 
+
+def setup_kubernetes_version(skuba, kubectl, kubernetes_version=None):
+    """
+    Initialize the cluster with the given kubernetes_version, bootstrap it and
+    join nodes.
+    """
+
+    skuba.cluster_init(kubernetes_version)
+    skuba.node_bootstrap()
+
+    skuba.join_nodes()
+
+    wait(check_nodes_ready,
+         kubectl,
+         wait_delay=60,
+         wait_backoff=30,
+         wait_elapsed=60*10,
+         wait_allow=(AssertionError))
+
+
+def create_skuba_config(kubectl, configmap_data, dry_run=False):
+    return kubectl.run_kubectl(
+        'create configmap skuba-config --from-literal {0} -o yaml --namespace kube-system {1}'.format(
+            configmap_data, '--dry-run' if dry_run else ''
+        )
+    )
+
+
+def replace_skuba_config(kubectl, configmap_data):
+    new_configmap = create_skuba_config(kubectl, configmap_data, dry_run=True)
+    return kubectl.run_kubectl("replace -f -", stdin=new_configmap.encode())
+
+
+def get_skuba_configuration_dict(kubectl):
+    skubaConf_yml = kubectl.run_kubectl(
+        "get configmap skuba-config --namespace kube-system -o jsonpath='{.data.SkubaConfiguration}'"
+    )
+    return yaml.load(skubaConf_yml, Loader=yaml.FullLoader)
+
+
+def decrease_one_addon_manifest(addons_dict, skip=None):
```

Comment:
I think it does, it just a shared util used in different tests. I could move it to a file called lib instead, but in the end in a shared, isolated function. Its specific? Yeah it is, but it could be reused in the future in other tests so having it centralized makes sense to me